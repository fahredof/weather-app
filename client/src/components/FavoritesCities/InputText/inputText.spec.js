import React from 'react';
import InputText from "./InputText";

describe(`Input`, () => {
    const props = {
        onChangeInput: () => {},
        value: undefined,
        onClickAddCity: () => {}
    };

    describe(`пустая строка`, () => {
        const input = shallow(<InputText {...props} />);
        it (`отображается корректно`, () => {
            expect(shallowToJson(input)).toMatchSnapshot();
        });
    });

    describe(`строка с названием города`, () => {
        const nextProps = {
            ...props,
            value: `Ufa`
        };
        const input = shallow(<InputText {...nextProps} />);
        it (`отображается корректно`, () => {
            expect(shallowToJson(input)).toMatchSnapshot();
        });
    });
});