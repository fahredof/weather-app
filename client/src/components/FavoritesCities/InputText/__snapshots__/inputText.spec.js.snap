// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input пустая строка отображается корректно 1`] = `
<div
  id="input"
>
  <form>
    <input
      onChange={[Function]}
      type="text"
    />
    <button
      onClick={[Function]}
      type="submit"
    >
      Add
    </button>
  </form>
</div>
`;

exports[`Input строка с названием города отображается корректно 1`] = `
<div
  id="input"
>
  <form>
    <input
      onChange={[Function]}
      type="text"
    />
    <button
      onClick={[Function]}
      type="submit"
    >
      Add
    </button>
  </form>
</div>
`;
